airflow/docker.go

```diff
@@ -568,6 +574,58 @@ func (d *DockerCompose) getWebServerContainerID() (string, error) {
 	return "", err
 }
 
+func (d *DockerCompose) RunTest(dagID, settingsFile, startDate string, noCache, taskLogs bool) error {
+	// Get project containers
+	psInfo, err := d.composeService.Ps(context.Background(), d.projectName, api.PsOptions{
+		All: true,
+	})
+	if err != nil {
+		return errors.Wrap(err, composeCreateErrMsg)
+	}
+	if len(psInfo) > 0 {
+		// Ensure project is not already running
+		for i := range psInfo {
+			if checkServiceState(psInfo[i].State, dockerStateUp) {
+				if strings.Contains(psInfo[i].Name, SchedulerDockerContainerName) {
+					err = d.imageHandler.RunTest(dagID, d.envFile, settingsFile, startDate, psInfo[i].Name, taskLogs)
+					if err != nil {
+						return err
+					}
+					return nil
+				}
+			}
+		}
+	}
+
+	fmt.Println("Building image... For a faster 'astro run' experince run this command while Airflow is running with 'astro dev start'\n ")
+	// add astro-run-dag
+	err = fileutil.AddLineToFile("./requirements.txt", "astro-run-dag", "# This package is needed for the astro run command. It will be removed before a deploy")
+	if err != nil {
+		fmt.Printf("Adding 'astro-run-dag' package to requirements.txt unsuccessful: %s\nManually add package to requirements.txt", err.Error())
+	}
+	// add airflow db init
+	err = fileutil.AddLineToFile("./Dockerfile", "RUN airflow db init", "")
+	if err != nil {
+		fmt.Printf("Adding line 'RUN airflow db init' to Dockerfile unsuccessful: %s\nYou may need to manually add this line for 'astro run' to work", err.Error())
+	}
+
+	err = d.imageHandler.Build(airflowTypes.ImageBuildConfig{Path: d.airflowHome, Output: true, NoCache: noCache})
+	// remove airflow db init
+	fileErr := fileutil.RemoveLineFromFile("./Dockerfile", "RUN airflow db init", "")
```

Comment:
should we check if the build succeeded before removing the line?

Reply:
My thought was to remove the line no matter what so the line doesn't confuse the user. If the build fails the line would still be in the Dockerfile. It always be removed